build.sbt

```diff
@@ -91,3 +91,18 @@ lazy val `bson-adt-mongo` = (project in file("bson-adt-mongo3")).settings(common
 ).dependsOn(
     `bson-adt-core` % "compile->compile;test->test"
   )
+
+lazy val `bson-adt-mongo-async` = (project in file("bson-adt-mongo3-async")).settings(common).settings(
+  name := "bson-adt-mongo-async",
+  libraryDependencies ++= Seq(
+    "org.mongodb" % "mongodb-driver-core" % "3.0.2",
+    "org.mongodb" % "mongodb-driver-async" % "3.0.2"
+  ),
+  testOptions in Test += Tests.Cleanup {
+    (loader: java.lang.ClassLoader) =>
+      println("Running test cleanup for bson-adt-mongo-async project...")
+      loader.loadClass("adt.bson.test.Cleanup").newInstance
```

Comment:
This was a pretty cool trick to clean up a singleton test client after the tests finished running.